diff --git a/dist/index.js b/dist/index.js
index 82df4b494e94d609818611917f7d50a76f3d8198..8db60b56efaec2114740d0f94bd535fe77428620 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -114,7 +114,7 @@ const VideoPlayer = (tempProps) => {
                     ? PlaybackStates.Ended
                     : status.isBuffering
                         ? PlaybackStates.Buffering
-                        : status.shouldPlay
+                        : status.isPlaying || status.shouldPlay
                             ? PlaybackStates.Playing
                             : PlaybackStates.Paused }));
             if ((status.didJustFinish && controlsState === ControlStates.Hidden) ||
