diff --git a/lib/src/components/BaseToast.js b/lib/src/components/BaseToast.js
index eb581d8329049ae8b6a4c3ca3c037bd9267226c2..abd3288d5b2879baa45ea1823ca939c36895db1a 100644
--- a/lib/src/components/BaseToast.js
+++ b/lib/src/components/BaseToast.js
@@ -3,14 +3,14 @@ import { Text, View } from 'react-native';
 import { getTestId } from '../utils/test-id';
 import { styles } from './BaseToast.styles';
 import { Touchable } from './Touchable';
-export function BaseToast({ text1, text2, onPress, activeOpacity, style, touchableContainerProps, contentContainerStyle, contentContainerProps, text1Style, text1NumberOfLines = 1, text1Props, text2Style, text2NumberOfLines = 1, text2Props, renderLeadingIcon, renderTrailingIcon }) {
+export function BaseToast({ text1, text2, onPress, activeOpacity, style, touchableContainerProps, contentContainerStyle, contentContainerProps, text1Style, text1NumberOfLines = 0, text1Props, text2Style, text2NumberOfLines = 0, text2Props, renderLeadingIcon, renderTrailingIcon }) {
     return (<Touchable testID={getTestId('TouchableContainer')} onPress={onPress} activeOpacity={activeOpacity} style={[styles.base, styles.leadingBorder, style]} {...touchableContainerProps}>
       {renderLeadingIcon && renderLeadingIcon()}
       <View testID={getTestId('ContentContainer')} style={[styles.contentContainer, contentContainerStyle]} {...contentContainerProps}>
-        {(text1?.length ?? 0) > 0 && (<Text testID={getTestId('Text1')} style={[styles.text1, text1Style]} numberOfLines={text1NumberOfLines} ellipsizeMode='tail' {...text1Props}>
+        {(text1?.length ?? 0) > 0 && (<Text testID={getTestId('Text1')} style={[styles.text1, text1Style]} numberOfLines={text1NumberOfLines} {...text1Props}>
             {text1}
           </Text>)}
-        {(text2?.length ?? 0) > 0 && (<Text testID={getTestId('Text2')} style={[styles.text2, text2Style]} numberOfLines={text2NumberOfLines} ellipsizeMode='tail' {...text2Props}>
+        {(text2?.length ?? 0) > 0 && (<Text testID={getTestId('Text2')} style={[styles.text2, text2Style]} numberOfLines={text2NumberOfLines} {...text2Props}>
             {text2}
           </Text>)}
       </View>
diff --git a/lib/src/components/BaseToast.styles.js b/lib/src/components/BaseToast.styles.js
index 105748bc5b96d0c1d2e4ccebad6f924da49c1d2c..5440cea463bf719a0cdf3212705de65ad11e0ca9 100644
--- a/lib/src/components/BaseToast.styles.js
+++ b/lib/src/components/BaseToast.styles.js
@@ -5,7 +5,7 @@ export const BORDER_RADIUS = 6;
 export const styles = StyleSheet.create({
     base: {
         flexDirection: 'row',
-        height: HEIGHT,
+        minHeight: HEIGHT,
         width: WIDTH,
         borderRadius: BORDER_RADIUS,
         shadowOffset: { width: 0, height: 0 },
